website/docs/sdks/unleash-proxy.md

```diff
@@ -41,46 +41,73 @@ Regardless of how you choose to run the it, the proxy will need access to these
 
   When using an environment variable to set the proxy secrets, the value should be a comma-separated list of strings, such as `secret-one,secret-two`.
 
-For more information, check out the [documentation on GitHub](https://github.com/Unleash/unleash-proxy).
+There are many more configuration options available. You find all [available options on github](https://github.com/Unleash/unleash-proxy#available-options).
+
 
 ### Running the proxy via Docker
 
 The easiest way to run Unleash is via Docker. We have published a [docker image on docker hub](https://hub.docker.com/r/unleashorg/unleash-proxy).
 
-**Step 1: Pull**
+1. **Pull the Proxy image**
+   ```bash
+   docker pull unleashorg/unleash-proxy
+   ```
```

Comment:
Just out of interest, do we need the pull step? A "run" should pull behind the scenes if there's no local image right?

Reply:
It's the recommended way, it'll only pull if there's no local image unless the user runs with `--always-pull`, so for making sure that they run with the latest image, remembering to pull is a good idea :)

Reply:
Fair enough! Apparently I'm terribly lazy when it comes to docker and I should stop being lazy

Reply:
Cool! I didn't know that either. I guess the option is to add the `--always-pull` flag as we've done a couple other places?